template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Program.Designer.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Company.WebApplication1
+{
+    public class DesignTimeServiceProviderFactory : IDesignTimeServiceProviderFactory
```

Comment:
Yeah I agree this is a bit weird... file name doesn't match type name, which I think makes this a bit *too* hard to find.